scipy/interpolate/_bsplines.py

```diff
@@ -1165,7 +1165,9 @@ def make_interp_spline(x, y, k=3, t=None, bc_type=None, axis=0,
     --------
 
     Use cubic interpolation on Chebyshev nodes:
+
     >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
```

Comment:
This is based on this [rule](https://github.com/scipy/scipy/issues/13049#issuecomment-1199457608).